.github/workflows/actions.yml

```diff
@@ -25,30 +15,6 @@ jobs:
         uses: dell/common-github-actions/code-sanitizer@main
         with:
           args: /github/workspace
-  test:
-    permissions: write-all
-    name: Run Go unit tests and check package coverage
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout the code
-        uses: actions/checkout@v4
-      - uses: actions/setup-go@v5
-        with:
-          go-version: "1.23"
-      - run: cd repctl && make test
```

Comment:
Should we keep the 'repctl' tests in this GitHub action, or consider including it along with all the tests?
@lukeatdell Your thoughts please.

Reply:
makes sense to move it, but based on the common actions, we can only specify a folder to test now, so these make targets won't be useful for actions.